src/components/Popups/ReplyField/ReplyField.scss

```diff
@@ -2,6 +2,10 @@
 @import '~draft-js/dist/Draft.css';
 
 .ba-ReplyField {
+    * {
```

Comment:
We should avoid `*` selectors at all times. CSS selectors are evaluated right-to-left, so this selects every element on the page (which is "slow") before filtering further.